### PR TITLE
Add german translation for cut wizard

### PR DESCRIPTION
**Pull request by marionb**
Please review.

### CODE REVIEW

**Comment by sbrunner** [comment]
This should also be added in french ...

**Comment by marionb** [comment]
I also added the translation to French, please review them.

File: core/src/script/CGXP/locale/de.js [hunk 1]
```diff
@@ -157,6 +157,11 @@ GeoExt.Lang.add("de", {
         undoButtonTooltip: "Die letzte Modifikation rückgängig machen STRG+Z",
         copyToBtnText: "Kopieren nach",
         copyToBtnTooltip: "Erstelle neues Feature in einem anderen Layer mit der gleichen Geometrie",
+	cutActionText: "Geometire zuschneiden",
```
Comment (sbrunner) [inline core/src/script/CGXP/locale/de.js]: wrong indent ...

File: core/src/script/CGXP/locale/fr.js [hunk 2]
```diff
@@ -161,6 +161,11 @@ GeoExt.Lang.add("fr", {
         undoButtonTooltip: "Annuler la dernière modification Ctrl+Z",
         copyToBtnText: "Copier vers",
         copyToBtnTooltip: "Créer un nouvel objet dans une autre couche avec la même géométrie",
+	cutActionText: "Couper la géométrie",
```
Comment (sbrunner) [inline core/src/script/CGXP/locale/fr.js]: also ...

File: core/src/script/CGXP/locale/fr.js [hunk 3]
```diff
@@ -161,6 +161,11 @@ GeoExt.Lang.add("fr", {
...
+        cutWizardTitle: "Couper le polygone",
+        cutWizardSubtitle: "Choisir le polygone B:",
```
Comment (asaunier) [inline core/src/script/CGXP/locale/fr.js]: "Choisir le polygone B :" (blank space before :)

**Comment by sbrunner** [comment]
Just indent issue, fix them and merge :-)

File: core/src/script/CGXP/locale/de.js (hunk 1 above)
Comment (tsauerwein) [inline core/src/script/CGXP/locale/de.js]: "Geometrie zuschneiden"

**Pull request merged**